owlbot.py

```diff
@@ -29,7 +29,7 @@
 for library in s.get_staging_dirs("v1"):
     # rename to google-cloud-recaptcha-enterprise
     s.replace(
-        [library / "google/**/*.py", library / "tests/**/*.py"],
+        [library / "google/**/*.py"],
```

Comment:
I remove `tests` here because I guess this is this what generates the MANIFEST file, but I'm not used to this tool, so it may be a mistake :)